projects/aca-content/src/lib/components/search/search-results/search-results.component.scss

```diff
@@ -1,4 +1,4 @@
-@import 'mixins';
+@import 'projects/aca-content/src/lib/ui/mixins';
```

Comment:
The same as [above](https://github.com/Alfresco/alfresco-content-app/pull/2817/files#r1052649093)